Hippo.Core/Domain/Request.cs

```diff
@@ -17,10 +17,8 @@ public class Request
         public int RequesterId { get; set; }
         public User Actor { get; set; }
         public int? ActorId { get; set; }
-        public Group Group { get; set; }
-        public int? GroupId { get; set; }
-        public Account Account { get; set; }
-        public int? AccountId { get; set; }
+        [MaxLength(32)]
+        public string Group { get; set; }
```

Comment:
if the sync happens and a group gets deleted, won't this prevent that?  i'm guessing it would throw an FK error when trying to delete and stop the whole bulk delete?

Reply:
Nope. It's not a foreign key. I probably should have named it `GroupName`. Queries referencing it have been updated to perform a left join to get at the associated group.